docs: LSDV-5343: Docs for visibleWhen; remove FF_DEV_1372

robot-ci-heartex opened the pull request:
Hi @hlomzik!

This PR was created in response to a PRs in upstream repo:
- https://github.com/heartexlabs/label-studio-frontend/pull/1483
- https://github.com/HumanSignal/label-studio-frontend/pull/1483

hlomzik committed:
[submodules] Build static heartexlabs/label-studio-frontend

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5379017878

hlomzik committed:
docs: LSF Update

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5379017878

hlomzik committed:
[submodules] Build static heartexlabs/label-studio-frontend

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5379047192

hlomzik committed:
[submodules] Build static heartexlabs/label-studio-frontend

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5383223119

hlomzik committed:
Merge branch 'origin/develop' into 'fb-lsdv-5343/visible-when'

hlomzik committed:
[submodules] Build static heartexlabs/label-studio-frontend

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5383578965

hlomzik commented:
/git merge develop
> Already up-to-date. Nothing to commit.
>
> [Workflow run](https://github.com/heartexlabs/label-studio/actions/runs/5383641494)

hlomzik committed:
[submodules] Build static heartexlabs/label-studio-frontend

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5423103898

hlomzik committed:
Merge branch 'origin/develop' into 'fb-lsdv-5343/visible-when'

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5423103898

hlomzik committed:
[submodules] Build static heartexlabs/label-studio-frontend

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5423889083

hlomzik committed:
[submodules] Build static heartexlabs/label-studio-frontend

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5449814349

hlomzik committed:
Merge branch 'origin/develop' into 'fb-lsdv-5343/visible-when'

Workflow run: https://github.com/heartexlabs/label-studio/actions/runs/5449814349

hlomzik committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5880811896

hlomzik committed:
Merge branch 'origin/develop' into 'fb-lsdv-5343/visible-when'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5880811896

hlomzik committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5890902627

hlomzik committed:
docs: LSF Update

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5890902627

hlomzik committed:
Merge branch 'origin/develop' into 'fb-lsdv-5343/visible-when'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5890902627
